Jenkinsfile

```diff
@@ -909,7 +909,7 @@ pipeline{
 		parallel{
 		 stage('Core-Unit-Tests'){
 		 agent { label 'dhfLinuxAgent'}
-			steps{Tests()}
+			steps{fullCycleSingleNodeTestOnLinux('Release', '10.0-10')}
```

Comment:
This needs to be reverted before merging. We just want to see how Jenkins reacts before merging.